setup.py

```diff
@@ -94,19 +94,6 @@
         except ImportError:
             s = (modname + ' '*25)[:25]
             missing.append('     %s %s' % (s, desc))
-missing_reqs = []
-for mod in modules_imported:
-    if mod in required_modules and not modules_imported[mod]:
-        missing_reqs.append(mod)
-
-if len(missing_reqs) > 0:
-    print('\n=== Cannot Install Larch, these REQUIRED Modules are missing: ')
-    print(' %s' % (', '.join(missing_reqs)))
-    print(' ')
-    print(' Please read INSTALL for further information.')
-    print(' ')
-    sys.exit()
-
```

Comment:
Please restore this. 
We still want to support building with setup.py, and we want this explicit message.

Reply:
`sys.exit()` will cause `pip install xraylarch` to fail since it's called before `setup()`, so no dependencies will install automatically.

Actually, I tried to install it on a fresh & clean conda environment with `python setup.py install` from this branch, and it did install all the dependencies listed in requirements.txt. I can provide you a log of installation if you wish.

Reply:
Maybe I'm missing some use cases though. Which commands do you execute with `python setup.py ...`?

Reply:
Well,  sometimes (or maybe often ;) we do use `python setup.py build` and `install`.  ;).

My concern here is that  `pip install` automatically installing packages might not be what we want (that is, preferring `conda`).  But, it probably is fine to automatically pip install these "hard" requirements (as opposed to some of the "optional packages" -- which seem to be the problems for `pip install`).

So, OK, you've convinced me to just `pip install` the missing packages in `requirements.txt`.

Reply:
Great! Any other concerns?